core/utils/dom.js

```diff
@@ -273,7 +273,7 @@ Blockly.utils.dom.getTextWidth = function(textElement) {
  * This method requires that we know the text element's font family and size in
  * advance. Similar to `getTextWidth`, we cache the width we compute.
  * @param {!Element} textElement An SVG 'text' element.
- * @param {number} fontSize The font size to use.
+ * @param {string} fontSize The font size to use.
```

Comment:
This looks like a breaking change, so it should be marked for doc updates.

Reply:
The change here is just whether you expect the 'pt' on the font size to be added here or by the calling code, right?  What's the advantage of changing it?

Reply:
The flyout button dynamically reads from CSS what the font size should be, which generates the font size in pixels. I needed to change this in order to support both ``pt`` for fields and ``px`` for flyout buttons. 

As I think about this, there's a way to do this without making it a breaking change. I'll just add another util method that uses pixels.

Reply:
I'm struggling to come up with the other method name, how do we feel about ``getFastTextWidthWithSizeString``.